lib/iex/lib/iex/helpers.ex

```diff
@@ -643,4 +644,42 @@ defmodule IEx.Helpers do
              Integer.to_char_list(z) ++ '>'
     )
   end
+
+  @doc """
+  Deloys a given module's beam code to a list of connected nodes.
+
+  This function is useful for development and debugging when you have code that
+  has been compiled or updated locally that you want to run on other nodes.
+  
+  The node list defaults to a list of all connected nodes.
+
+  Returns `{:error, :nofile}` if the object code (i.e. ".beam" file) for the module
+  could not be found locally.
+
+  ## Examples
+
+      nl(HelloWorld)
+      #=> {:ok, [{:node1@easthost, :loaded, HelloWorld},
```

Comment:
Hi @lexmag. Interesting point.

In the actual output there is no indentation or line-breaks. The return is just a data structure. So the true output would be:  
```
{:ok, [{:node1@easthost, :loaded, HelloWorld}, {:node1@westhost, :loaded, HelloWorld}]}
```
My thinking was the line-breaks and indentation will make it more clear to the reader that for each node there is a response tuple. 

Happy to modify it to the actual output (possibly less effective doc), or to your suggested output (which I like better than my original, but isn't _actual output_ either). 

What do you think is best?